testing/full-integration.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cd $(dirname "$0")
```

Comment:
untested but maybe try something like `cd "$(dirname "$0")" || exit 1` to cleanup code climate issues